modules/ethereum/src/lib.rs

```diff
@@ -1225,7 +1261,7 @@ pub(crate) mod tests {
 
 	#[test]
 	fn verify_transaction_finalized_works_for_best_finalized_header_ancestor() {
-		custom_test_ext(genesis(), validators_addresses(3)).execute_with(|| {
+		run_test(3, |_| {
```

Comment:
The number passed to `run_test` is a bit cryptic. Maybe we could extract it to a constant to give it some more meaning, also given that it repeats for mulitple tests.